common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -205,16 +211,6 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
     }
 
     @Override
-    public Iterable<Entry<V>> entries() {
```

Comment:
> I think it's fair to assume that iteration over a map is slow :)

Are we adding boxing penalties on top of the iteration though?

> With that said, I think the places where it's done aren't frequently called or terribly time-sensitive.

One interesting place where we iterate over the entries is in `DefaultHttp2RemoteFlowController` every time we allocate bytes. It would be good to understand the impact in this case.